ChaosMod/Memory/Drawing.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include "Util/Color.h"
+
+// Thanks CitizenFX!
+// Adapted by MoneyWasted for ChaosModV
+// https://github.com/citizenfx/fivem/blob/master/code/components/extra-natives-five/src/Draw2dNatives.cpp
+
+// This file is manually formatted.
+// clang-format off
+
+constexpr int MAX_LINES = 5000;
+
+char *(*OG_AllocateDrawRect)(void *);
+void (*OG_SetDrawRectCoords)(void *, float, float, float, float);
```

Comment:
No need for those, you can simply use local static variables (and drop the redundant OG_ prefix)